sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -392,7 +393,7 @@ export class MessageSession extends LinkEntity {
           log.error("[%s] %O", this._context.namespace.connectionId, error);
           throw error;
         }
-        if (this.sessionId == null) this.sessionId = receivedSessionId;
+        if (this.sessionId == undefined) this.sessionId = receivedSessionId;
```

Comment:
This change is not needed right?

Reply:
Yes, not needed, thought I'd leave it to be consistent.

Reverted now.